jsapp/js/account/usage/usage.api.ts

```diff
@@ -73,3 +72,40 @@ export async function getUsage(organization_id: string | null = null) {
     errorMessageDisplay: t('There was an error fetching usage data.'),
   });
 }
+
+export async function getAssetUsage(url?: string) {
+  const apiUrl = url || ASSET_USAGE_URL;
+  return fetchGet<AssetUsage>(apiUrl, {
+    includeHeaders: true,
+    errorMessageDisplay: t('There was an error fetching asset usage data.'),
+  });
+}
+
+export async function getAssetUsageForOrganization(
+  pageNumber: number,
+  order?: ProjectsTableOrder
+) {
+  let organizations;
+  try {
+    organizations = await getOrganization();
+  } catch (error) {
+    return await getAssetUsage(ASSET_USAGE_URL);
+  }
+
+  let apiUrl = ORGANIZATION_ASSET_USAGE_URL.replace(
+    '##ORGANIZATION_ID##',
```

Comment:
I'd probably define the urls as functions rather than use a custom string token to be replaced. But this was done outside of this PR, so out of scope. I'm not sure what is most popular for react devs to do.